### PR TITLE
sql server support

### PR DESCRIPTION
**Pull request by andyliuliming**
Thanks for contributing to cloud_controller_ng. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
sql server support for ccdb

* An explanation of the use cases your change solves

* [*] I have viewed signed and have submitted the Contributor License Agreement

* [*] I have made this pull request to the `master` branch

* [*] I have run all the unit tests using `bundle exec rake`

* [*] I have run CF Acceptance Tests on bosh lite

### CODE REVIEW

**Comment by andyliuliming** [comment]
@Gerg Hi Gerg, I do not have permission to open your codeclimate, could you please help show me the new issue? I did a codeclimate on my repository, no luck to find the new issue. Thanks.

File: app/controllers/runtime/syslog_drain_urls_controller.rb [hunk 1]
```diff
@@ -10,20 +10,29 @@ class SyslogDrainUrlsController < RestController::BaseController
```
Comment (bingosummer) [inline app/controllers/runtime/syslog_drain_urls_controller.rb]: The permission of the file is changed from `100644` to `100755`. Is it by design?

Comment (andyliuliming) [inline app/controllers/runtime/syslog_drain_urls_controller.rb]: resolved :) Thanks.

**Comment by SocalNick** [comment]
@andyliuliming supporting SQL Server is currently not a priority for the CAPI team. We currently run unit and integration tests against MySQL and PostgreSQL, this would expand our testing matrix and we aren't prepared to take that on at the current time.

**Comment by NingKuang** [comment]
Hi, @SocalNick 
Seems to me it is the intention of open source Cloud Foundry to support multiple databases based customer's needs. 
Do you have a timeframe or a plan to support other databases in the future? Or are you open to other supporting models, like allowing the community to contribute to the test and support ?
Thanks.

**Comment by MarkKropf** [comment]
Hello @NingKuang,

I've spoken to the dev team about this, we're taking another look at the PR and should have an updated response soon. I definitely see the value in supporting MSSQL, thank you for taking the time to submit this. 

Regards,

Mark

**Comment by ljfranklin** [comment]
To avoid confusion, I'm going to close this PR in favor of a new PR that points to the current SQL Server branch: https://github.com/cloudfoundry/cloud_controller_ng/pull/802

**Pull request closed without merging**